robot/Crisis-Management/resources/locators_48.py

```diff
@@ -71,6 +71,9 @@
         "home_tab": "//a[@href='/TractionThrive/s/']",
         "staff_info": "//div/lightning-formatted-text[text()='{}']",
         "staff_contact_info": "//lightning-input[@data-field='{}']//input",
+        "user_info": "//div[contains(@title, '{}')]",
+        "user_sub_info": "//a[contains(@title, '{}')]",
```

Comment:
you could do just this `"user_info": "//*[contains(@title, '{}')]"` since the only thing that differs here is the tags(div ,a) and reuse the same locator for both `name & My Account`

Reply:
Done.